### ISSUE TITLE
question

### ISSUE THREAD

**Issue by DiSheng555** [xytian1008/ArGue#5]
Prompt 一共可以训练参数多少啊，是不是类别*prompt?

**Comment by HUOUO7** [comment]
I think each category has multiple attributes of learnable prompt vectors, so the total number of learnable vectors = number of categories * number of attributes after clustering of each category, so it seems that there are still many trainable parameters.

**Comment by StellarOdys2ey** [comment]
> I think each category has multiple attributes of learnable prompt vectors, so the total number of learnable vectors = number of categories * number of attributes after clustering of each category, so it seems that there are still many trainable parameters.

That's not correct. There are only 4 learnable parameters for all the categories and their corresponding attributes, following the settings of CoOp. This means optimizing a unified soft prompt for all texts.
